tests/etl/test_load_artifacts.py

```diff
@@ -72,6 +73,99 @@ def test_load_non_ascii_textlog_errors(test_job):
     assert TextLogError.objects.get(line_number=1588).line == "07:51:29  WARNING - <U+01D400>"
 
 
+def _add_job_summary(job, expected_new):
+    text_log_summary_artifact = {
+        "type": "json",
+        "name": "text_log_summary",
+        "blob": json.dumps(
+            {
+                "errors": [
+                    {
+                        "line": "TEST-UNEXPECTED-FAIL | mypath/mytest.js | unexpected value 3.14",
+                        "linenumber": 1587,
+                    },
+                ],
+            }
+        ),
+    }
+
+    # ensure a result='failed' to treat failure as a NEW_failure
+    job.result = "testfailed"
+    job.save()
+
+    text_log_summary_artifact["job_guid"] = job.guid
+    store_job_artifacts([text_log_summary_artifact])
+
+    # ensure bug_suggestions data is stored and retrieved properly
+    tle_all = TextLogError.objects.all()
+    bug_suggestions = get_error_summary(job)
+    for suggestion in bug_suggestions:
+        tle = next(t for t in tle_all if t.line_number == suggestion["line_number"])
```

Comment:
`tle` in line 104 overwrites the one from line 103.

Reply:
thanks!